annotation_widgets/event_validation/models.py

```diff
@@ -0,0 +1,59 @@
+import json
+from typing import List
+
+from sqlalchemy import asc, Column, String, Integer
+
+from db import Base, get_session
+
+
+class Event(Base):
```

Comment:
While testing i found that the table Events contains rows with duplicated item_id.
adding `unique=True` for item_id is not a comprehensive solution
It is better to remove Event.item_id field and use only Event.uid 

Store sorted list of event video names in EventValidationLogic
In `EventValidationLogic.load_item` get a name by id from this list, then get uid from a name, and then get Event by this uid